endless-sky.6

```diff
@@ -44,6 +44,9 @@ execute the test case with the given name
 .IP \fB\-\-tests
 prints (to STDOUT) a table of available tests, usable for automatic test runs. This option prevents the game from launching.
 
+.IP \fB\-\-list\-tests
+prints (to STDOUT) a list of every runnable test, usable for automatic test runs. This option prevents the game from launching.
```

Comment:
Why not re-use the output from `--tests`? Should be a 2 lines change in the shell-script, the shell script already filters on "known failure" and on "missing feature"; just move those to the OK category.

Reply:
It's for CMake: It's cause I do not want to filter the table using CMake script (and calling an external script to do so is also not clean).

Reply:
Ok, but then I still suggest to replace `--tests` and make it return the output you want it to give. We don't need 2 functions to list tests on the main executable of the game.